### DISCUSSION TITLE
Show viewed files count on the PR. Publish only if reviewer chooses the option..

### DISCUSSION THREAD

**Discussion by kaladhar-mummadi** [community/community#39861] [Pull Requests]
### Select Topic Area

Product Feedback

### Description
If PR reviewer chooses to publish his review progress on the page as status then show it as count ( User1: 5/18 viewed )
The option can be a toggle per user. Something like  "🔘 Publish/Show my `Viewed` count"
####  or
 we show `92/99 files viewed` as hyperlink which opens a dialog to toggle the publish option.
<img width="346" alt="Screenshot 2022-11-23 at 9 21 54 AM" src="https://user-images.githubusercontent.com/3730954/203610205-21c42062-fc4d-4057-93fb-cf25a19a6bee.png">

**Comment by ossanna16** [comment]
Hi there @kaladhar-mummadi and welcome to our community! Thank you for the feedback!
